include/constants/moves.h

```diff
@@ -787,3 +787,9 @@
 #define MOVES_COUNT MOVES_COUNT_GEN8
 
 #endif  // GUARD_CONSTANTS_MOVES_H
+
+// moves renamed in BE
+#define MOVE FAINT_ATTACK MOVE_FEINT_ATTACK
+#define MOVE_HI_JUMP_KICK MOVE_HIGH_JUMP_KICK
+#define MOVE_SMELLING_SALT MOVE_SMELLING_SALTS
+#define MOVE_VICE_GRIP MOVE_VISE_GRIP
```

Comment:
```suggestion
```
Since Pokemon Expansion currently syncs its move defines from Battle Engine Upgrade, these additions are unnecessary and we can just merge a separate PR that syncs the new move defines in to Pokemon Expansion soon after.